enterprise/reporting/reporting.go

```diff
@@ -196,6 +196,10 @@ func (r *DefaultReporter) getReports(currentMs int64, syncerKey string) (reports
 	if err != nil {
 		panic(err)
 	}
+	if rows.Err() != nil {
```

Comment:
Same as the above comment. We need to move this once the loop for `rows.Next()` is completed.